src/Node.java

```diff
@@ -0,0 +1,40 @@
+import java.util.ArrayList;
+
+public class Node {
+
+    private boolean isFinal;
+    private final char value;
+    private ArrayList<Edge> edges = new ArrayList<>();
+
+    Node(char value) {
+        this.value = value;
+    }
+
+    public boolean isFinal() {
+        return isFinal;
+    }
+
+    public void setFinal(boolean aFinal) {
+        isFinal = aFinal;
+    }
+
+    public char getValue() {
+        return value;
+    }
+
+    public ArrayList<Edge> getEdges() {
+        return edges;
+    }
+
+    public void addEdge(Edge edge) {
+        edges.add(edge);
+    }
```

Comment:
This is a bit controversial as it implies you can't add to the list being returned above (which is not true). I guess the proper way would be to return an unmodifiable collection from `getEdges` and expose adding with this method.
Implementing your own `add` makes sense in order to prevent adding the same letter edge more than once. This may be left for a future change along with the tests which will expose all kinds of edge cases (pun not intended).